package.json

```diff
@@ -52,8 +52,8 @@
     "url": "https://github.com/code-mancers/interceptor.git"
   },
   "scripts": {
-    "build": "webpack -p --config webpack.config.prod.js",
-    "watch": "webpack --config webpack.config.dev.js --watch --progress --colors",
+    "build": "./node_modules/.bin/webpack -p --config webpack.config.prod.js",
```

Comment:
Why this change? npm script will find the executable from `./node_modules/.bin/`. no need to specify this.

Reply:
Okay, will revert it back. Didn't know about this ^^